src/utilities.jl

```diff
@@ -145,3 +145,71 @@ end
 
 check_positive_definite(M::AbstractMatrix; kwargs...) =
   check_positive_definite(LinearOperator(M); kwargs...)
+
+
+  """
+  solve_shifted_system!(B, 
+                        z, 
```

Comment:
Where did z come from? It’s b everywhere else. And I guess the argument order should be
```julia
solve_shifted_system!(x, B, b, σ)
```